molecule/local/molecule.yml

```diff
@@ -20,8 +19,8 @@ scenario:
     - destroy
 lint: |
   set -e
-  yamllint . || true
-  ansible-lint || true
+  yamllint .
+  ansible-lint
```

Comment:
Can we run these commands with `./run-qa-checks` so that developers validating QA checks locally can spot lint errors before they push or in any case they'd get the same errors locally if the CI fails?